doc/source/cluster/kubernetes/user-guides/gcp-gke-tpu-cluster.md

```diff
@@ -0,0 +1,52 @@
+(kuberay-gke-tpu-cluster-setup)=
+
+# Start Google Cloud GKE Cluster with TPUs for KubeRay
```

Comment:
It would be awesome if this guide show-cased a real example of using TPUs beyond just creating a GKE cluster with KubeRay and  TPUs enabled

Reply:
Added a Stable Diffusion example in[4296124](https://github.com/ray-project/ray/pull/45924/commits/42961243c485652ae4d35de1aa3779f12b199aed).